src/reducers/statusEffects.js

```diff
@@ -1,21 +1,26 @@
 import { BLEED_OUT } from '../actions/bleeding';
 import { FEEL_TIRED } from '../actions/tired';
 
+const singleStackEffects = [BLEED_OUT, FEEL_TIRED];
+const isSingleStackEffect = singleStackEffects.includes;
+
+const multiStackEffects = [];
+const isMultiStackEffect = multiStackEffects.includes;
+
 const initialStatusEffects = [];
 
-const isBleeding = state => state.find(effect => effect.type === BLEED_OUT);
-const isTired = state => state.find(effect => effect.type === FEEL_TIRED);
+const effectApplied = (state, type) => state.find(effect => effect.type === type);
 
 const statusEffects = (state = initialStatusEffects, action) => {
-  switch (action.type) {
-  // update the array of status effects to record the new effect:
-  // if already applied, do nothing; otherwise, apply action
-    case BLEED_OUT:
-      return isBleeding(state) ? state : [...state, action];
-    case FEEL_TIRED:
-      return isTired(state) ? state : [...state, action];
-    default: return state;
+  const singleStack = isSingleStackEffect(action.type);
+  const multiStack = isMultiStackEffect(action.type);
+  const isEffect = singleStack || multiStack;
+
+  if (!isEffect || (singleStack && effectApplied(state, action.type))) {
```

Comment:
If the action isn't an effect, or it's a single stack effect that's already applied, we ignore it.

Would need to change this if the desire was for the newly applied effect to override the old one, or for the greater of the two to apply, or whatevs, but that's just a new if statement if so.

Also, the GH comments in this file should probably be code comments ^^;